bindings/nodejs/lib/index.ts

```diff
@@ -5,7 +5,7 @@
 import 'reflect-metadata';
 
 export * from './client';
-export * from './secretManager';
+export * from './secret_manager';
```

Comment:
Or should this be kebab case?

Reply:
And secret_manager is a folder, so snake